textfinding_indikator1.py

```diff
@@ -0,0 +1,120 @@
+# this code is for TEXT FINDING - INDIKATOR 1
+# input is txt file from preprocess dokumen lama, nama instansi & judul from dokumen lama & baru
+
+import re
+import preprocess_dokbaru as dokbaru
+import preprocess_doklama as doklama
+from utility import *
+
+
+def txtreader(filename, lv, keyword):
+    # func to search keyword in txt file
+
+    # open txt file
+    file = open(f'cleaned_{filename}.txt', 'r')
+
+    idx = 0
+    result = []
+    init_keywords = list(keyword)
```

Comment:
I think there's a little typo here. This one initialize `init_keywords` variable (with `s`), but `init_keyword` (without `s`) is the one that's being used throughout the file

Reply:
Thanks for the keen eye! I've fixed it according to the suggested changes. Please give it another review if you've time.